mopidy_spotify/playback.py

```diff
@@ -15,6 +15,9 @@
 # Extra log level with lower importance than DEBUG=10 for noisy debug logging
 TRACE_LOG_LEVEL = 5
 
+# Last audio data sent to the buffer, currently on hold
```

Comment:
I would put in a small explanation about why we need to do this, it's not obvious without the context of this PR. So perhaps a short sentence with a link to this PR or a self contained paragraph?